### PR TITLE
No result message bugfix

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
## Purpose
`NoResultsMessage` was displaying incorrectly on provider-package and package-title components.

### CODE REVIEW

File: src/components/query-list/query-list.js [hunk 1]
```diff
@@ -84,9 +83,9 @@ export default class QueryList extends Component {
               {state.rejected[0].error[0].title}
             </div>
           ) : !state.length ? (
-            <NoResultsMessage data-test-query-list-not-found={type}>
-              {notFoundMessage}
-            </NoResultsMessage>
+            <>
```
Comment (MaksymDryha) [inline src/components/query-list/query-list.js:86]: this fragment is unnecessary

File: src/components/query-list/not-found/query-not-found.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import classnames from 'classnames/bind';
+import styles from '../query-list.css';
+
+
+const cx = classnames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  type: PropTypes.string.isRequired,
```
Comment (MaksymDryha) [inline src/components/query-list/not-found/query-not-found.js:12]: `PropTypes.oneOf(['provider-packages', 'package-titles']).isRequired`

**Review by MaksymDryha — APPROVED** [review]

File: src/components/query-list/not-found/query-not-found.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const cx = classnames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  type: PropTypes.oneOf(['provider-packages', 'package-titles']).isRequired,
+};
+
+const QueryNotFound = ({ children, type }) => (
+  <div className={cx('error', 'not-found')} data-test-query-list-not-found={type}>
```
Comment (Vladyslav-Velytskyi) [inline src/components/query-list/not-found/query-not-found.js:16]: Please, set each prop in separate line

File: src/components/query-list/not-found/query-not-found.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import classnames from 'classnames/bind';
+import styles from '../query-list.css';
+
+
```
Comment (Vladyslav-Velytskyi) [inline src/components/query-list/not-found/query-not-found.js:7]: Please delete this empty line

File: src/components/package/show/package-show.js [hunk 5]
```diff
@@ -539,7 +540,10 @@ class PackageShow extends Component {
         length={packageTitles.length}
         scrollable={scrollable}
         itemHeight={ITEM_HEIGHT}
-        notFoundMessage={<FormattedMessage id="ui-eholdings.notFound" />}
+        notFoundMessage={
+          <QueryNotFound type="package-titles">
+            <FormattedMessage id="ui-eholdings.notFound" />
+          </QueryNotFound>}
```
Comment (Vladyslav-Velytskyi) [inline src/components/package/show/package-show.js:546]:
```suggestion
          </QueryNotFound>
        }
```

File: src/components/provider/show/provider-show.js [hunk 6]
```diff
@@ -267,7 +268,10 @@ class ProviderShow extends Component {
         length={packages.length}
         scrollable={scrollable}
         itemHeight={ITEM_HEIGHT}
-        notFoundMessage={<FormattedMessage id="ui-eholdings.notFound" />}
+        notFoundMessage={
+          <QueryNotFound type="provider-packages">
+            <FormattedMessage id="ui-eholdings.notFound" />
+          </QueryNotFound>}
```
Comment (Vladyslav-Velytskyi) [inline src/components/provider/show/provider-show.js:274]:
```suggestion
          </QueryNotFound>
        }
```

File: src/components/query-list/not-found/query-not-found.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  children: PropTypes.node.isRequired,
+  type: PropTypes.oneOf(['provider-packages', 'package-titles']).isRequired,
+};
+
+const QueryNotFound = ({ children, type }) => (
+  <div
+    className={cx('error', 'not-found')}
+    data-test-query-list-not-found={type}
+  >
+    { children }
```
Comment (Vladyslav-Velytskyi) [inline src/components/query-list/not-found/query-not-found.js:19]:
```suggestion
    {children}
```

File: src/components/query-list/query-list.js [hunk 8]
```diff
@@ -84,9 +83,7 @@ export default class QueryList extends Component {
               {state.rejected[0].error[0].title}
             </div>
           ) : !state.length ? (
```
Comment (Vladyslav-Velytskyi) [inline src/components/query-list/query-list.js:85]: In this case you can omit bracers
```suggestion
          ) : !state.length
            ? notFoundMessage
            : (
              <ScrollView
```

File: src/components/title/edit/title-edit.js [hunk 9]
```diff
@@ -95,7 +95,7 @@ export default class TitleEdit extends Component {
           mutators={{ ...arrayMutators }}
           render={({ handleSubmit, pristine, form: { reset } }) => (
             <>
-              <form onSubmit={handleSubmit}>
+              <form onSubmit={handleSubmit} noValidate>
```
Comment (Vladyslav-Velytskyi) [inline src/components/title/edit/title-edit.js:98]:
```suggestion
              <form
                onSubmit={handleSubmit}
                noValidate
              >
```

File: src/components/package/edit-custom/custom-package-edit.js [hunk 10]
```diff
@@ -290,7 +290,7 @@ class CustomPackageEdit extends Component {
         render={({ handleSubmit, pristine, form: { change, reset } }) => (
           <div>
             <Toaster toasts={processErrors(model)} position="bottom" />
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={handleSubmit} noValidate>
```
Comment (Vladyslav-Velytskyi) [inline src/components/package/edit-custom/custom-package-edit.js:293]:
```suggestion
            <form
              onSubmit={handleSubmit}
              noValidate
            >
```

**Review by Vladyslav-Velytskyi — APPROVED** [review]

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**